package.json

```diff
@@ -59,6 +59,7 @@
         "watch:labextension": "jupyter labextension watch ."
     },
     "dependencies": {
+        "@jupyter/chat": "^0",
```

Comment:
Can we provide a more meaning version number for this?

Reply:
However, the `jupyterlab-chat` package itself was not published to NPM until v0.6.0, so that will serve as the minimum version declared here.

Reply:
Note that the version specifier for `jupyterlab-chat` will look slightly different since it is still in the 0.x major version. Using `^0.6.0` or `~0.6.0` will only match `0.6.*` versions of `jupyterlab-chat`, so I will have to use `>=0.6.0 <1`.

From https://semver.npmjs.com/#syntax-examples:

<img width="961" height="570" alt="Image" src="https://github.com/user-attachments/assets/caa246e8-6302-4cb8-83b1-f5ea5d08d1b8" />

Reply:
This is now implemented in the latest commit. 👍